kfs-core/src/main/java/edu/arizona/kfs/fp/businessobject/ProcurementCardLevel3AddItem.java

```diff
@@ -0,0 +1,144 @@
+package edu.arizona.kfs.fp.businessobject;
+
+import java.math.BigDecimal;
+import java.util.LinkedHashMap;
+
+import org.kuali.rice.core.api.util.type.KualiDecimal;
+import org.kuali.rice.krad.bo.PersistableBusinessObjectBase;
+
+public class ProcurementCardLevel3AddItem extends PersistableBusinessObjectBase {
+
+	private String documentNumber;
+	private Integer financialDocumentTransactionLineNumber;
+	private Integer sequenceNumber;
+	private String itemCommodityCode;
+	private String itemProductCode;
+	private String itemDescription;
+	private BigDecimal itemQuantity;
```

Comment:
OK, with a little more research I found this. The property was changed from Integer to BigDecimal in the following commit: https://github.com/ua-eas/eas-ksd-kfs-3.0/commit/820b32dfd63ce4448425f4c9c9350e18cf29ea83#diff-fb0cb3986d756942f5196b72c66ff986. The reason is explained in Jira KITT-776 (VisaLine/itemQuantity - Value from JPMC can be a decimal).